test/lsp_test_runner.cc

```diff
@@ -417,27 +417,35 @@ void testDocumentFormatting(LSPWrapper &lspWrapper, Expectations &test, int &nex
                                                         make_unique<FormattingOptions>(4, 4));
     auto req = make_unique<RequestMessage>("2.0", nextId++, LSPMethod::TextDocumentFormatting, move(params));
     auto responses = getLSPResponsesFor(lspWrapper, make_unique<LSPMessage>(move(req)));
-    {
-        INFO("Did not receive exactly one response for a documentFormatting request.");
+    { INFO("Did not receive exactly one response for a documentFormatting request."); }
```

Comment:
our test harness is weird. These INFO messages are only shown if an assertion in the same block scope fails.

Before, if the `REQUIRE_EQ` failed, this `INFO` would be shown in the test error output. Now, this `INFO` output will never be shown iiuc.